src/hooks/useNodePos.tsx

```diff
@@ -15,8 +15,9 @@ export function NodePosProvider({ nodeKey, children }: Props) {
   const editorState = useEditorState();
   if (!editorState) return <>{children}</>;
   const pluginState = reactPluginKey.getState(editorState);
+  if (!pluginState) throw new Error('Can\'t find the react() ProseMirror plugin. Was it added to the EditorState.plugins?')
```

Comment:
How would you feel about moving this check [here](https://github.com/nytimes/react-prosemirror/blob/main/src/nodeViews/createReactNodeViewConstructor.tsx#L122-L124)? That would cover this case as well, and reduce the likelihood that someone runs into unexpected behavior re: context flow.